src/otaclient/_status_monitor.py

```diff
@@ -329,6 +331,15 @@ def _status_collector_thread(self) -> None:
             except queue.Empty:
                 time.sleep(self.min_collect_interval)
 
+    def _ota_status_logging_thread(self) -> None:
+        while True:
+            # when in active OTA, log OTA progress very <LOG_OTA_PROGRESS_INTERVAL>
+            if (
+                _cur_status := self._status
+            ) and _cur_status.ota_status == OTAStatus.UPDATING:
```

Comment:
How about to  add `OTAStatus.ROLLBACKING` too?

Reply:
`OTAStatus.ROLLBACKING` is just configuring the bootloader to directly switch slot, no changes to rootfs will occur on both slots, so the progress will be very quick(succeed or fail).

Another thing is, actually ROLLBACK is not used and not officially announced. The code is just there, but it is not expected to be used.